es-app/src/Gamelist.cpp

```diff
@@ -127,7 +137,7 @@ void parseGamelist(SystemData* system)
 			}
 
 			// Check whether the file's extension is allowed in the system
-			if (std::find(allowedExtensions.cbegin(), allowedExtensions.cend(), Utils::FileSystem::getExtension(path)) == allowedExtensions.cend())
+			if (i == 0 /*game*/ && std::find(allowedExtensions.cbegin(), allowedExtensions.cend(), Utils::FileSystem::getExtension(path)) == allowedExtensions.cend())
```

Comment:
Only be strict on type game FileData, without this check no <folder/> elements will be loaded into ES from the gamelist.